docusaurus/docs/cloud/projects/notifications.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Notifications
+displayed_sidebar: cloudSidebar
+description: View your notifications on Strapi Cloud.
+canonicalUrl: https://docs.strapi.io/cloud/projects/notifications.html
+sidebar_position: 3
+---
+
+# Notifications
+
+The **Notification center** is accessible from the top navigation bar from anywhere in Cloud dashboard.
```

Comment:
```suggestion
The **Notification center** is accessible from the top navigation of the Cloud dashboard.
```
Also, a couple of feedbacks:
- It wasn't super clear for me if the Notification center was the drop-down list shown in the screenshot, or if that was just a preview and that I could access the proper Notification center interface by clicking on the bell icon. It would probably worth it to clarify that in this introduction. I'll probably change the word "accessible" that implicates some kind of redirection. What do you think? 😊  
- It could be a good idea to mention clearly in the sentence that the list of notifications can be shown/displayed by clicking on the bell icon, and to add the icon directly in the sentence as well (you have examples of that in the user guide). The more precise & visual, the better 😄

Reply:
I'll also recommend adding the content of l.22 in this sort of introduction. Here's an example of how I would add the information:

"It displays a list of the latest notifications for all existing projects. Clicking on a notification card from the list will redirect you to the **Log details** of the corresponding deployment."

Reply:
> I'll probably change the word "accessible" that implicates some kind of redirection. What do you think? 😊
Sure 👍 I did some change, pls feel free to edit it as you wish.

> ..and to add the icon directly in the sentence as well (you have examples of that in the user guide)
You mean emoji right, not the original PNG icon? (I checked the User guide and this is what I understood, as there are some bulb 💡 emojis there).

> I'll also recommend adding the content of l.22 in this sort of introduction.
Agree, done.

Reply:
> It wasn't super clear for me if the Notification center was the drop-down
Probably that's also because the screenshot was not a bit incorrect (notifs. center window was positioned a bit far away from bell icon, it's not like this in prod). So I updated the screenshot as well.

Reply:
@philippkhon No I mean the original PNG/SVG icon of the bell that's used in the interface, so that users can easily see it in the documentation and then connect with what they see in the interface 😊